pages/linux/scriptreplay.md

```diff
@@ -0,0 +1,15 @@
+# scriptreplay
+
+> Output content of session files created by script command to standard output.
+
+- Replay {{path/to/session.out}} at the speed it was recorded:
+
+`scriptreplay {{path/to/timingfile}} {{path/to/session.out}}`
```

Comment:
```suggestion
`scriptreplay {{path/to/timing_file}} {{path/to/typescript}}`
```
The simpler the tokens are, the better. Here since the second positional argument is a typescript, calling it `typescript` makes it easier to understand. It is also better to separate multiple words with an underscore.

Reply:
@mebeim: the `script` page at https://github.com/tldr-pages/tldr/pull/2918#discussion_r275518979,references typescript files as `session.out`, should the companion `scriptreplay` page be kept consistent?

Reply:
@lbonanomi yeah that would probably make it more understandable, yes :slightly_smiling_face: